$OpenBSD: patch-src_yelp-transform_c,v 1.2 2009/10/11 17:31:07 ajacoutot Exp $

XXX add a workaround for:
Program received signal SIGSEGV, Segmentation fault.
[Switching to process 201, thread 0x7de7a000]
0x0d363e73 in xmlXPathFreeNodeSet (obj=0x8644ce60) at xpath.c:4081
4081               if ((obj->nodeTab[i] != NULL) &&

--- src/yelp-transform.c.orig	Sun Oct 11 19:20:18 2009
+++ src/yelp-transform.c	Sun Oct 11 19:20:40 2009
@@ -186,7 +186,7 @@ yelp_transform_release (YelpTransform *transform)
 	/* We might still have pending pops from the queue, so just
 	 * schedule transform_free.
 	 */
-	g_idle_add ((GSourceFunc) transform_free, transform);
+	// g_idle_add ((GSourceFunc) transform_free, transform);
     }
     g_mutex_unlock (transform->mutex);
 }
